### PR TITLE
Fill cluster name property for traces

### PR DESCRIPTION
**Pull request by pmm-sumo**
###### Description

`cluster` property was not filled for traces previously. This fixes it, using the new tracing taxonomy (so it goes to `k8s.cluster.name`)

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [x] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml [hunk 1]
```diff
@@ -12,5 +12,6 @@
 {{- $excludePodRegex := .Values.fluentd.logs.containers.excludePodRegex | quote }}
 {{- $excludeContainerRegex := .Values.fluentd.logs.containers.excludeContainerRegex | quote }}
 {{- $excludeHostRegex := .Values.fluentd.logs.containers.excludeHostRegex | quote }}
+{{- $clusterName := .Values.sumologic.clusterName | replace " " "-" }}
```
Comment (frankreno) [inline deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml:15]: To have consistency with logs and metrics, you should use this one as logs and metrics use this one - https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/templates/_helpers.tpl#L317

Comment (pmm-sumo) [reply deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml]: Fair point! I attempted doing that originally but chart had some errors while rendering, though now I see it was a simple mistake and the helper attribute works here. Thank you!

File: deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml [hunk 2]
```diff
@@ -12,5 +12,6 @@
...
\ No newline at end of file
+{{- $yamlFile | replace "processors.source.collector.replace" $_collector | replace "exporters.zipkin.url_replace" $endpoint | replace "processors.source.name.replace" $sourceName | replace "processors.source.category.replace" $sourceCategory | replace "processors.source.category_prefix.replace" $sourceCategoryPrefix | replace "processors.source.category_replace_dash.replace" $sourceCategoryReplaceDash | replace "processors.source.exclude_namespace_regex.replace" $excludeNamespaceRegex | replace "processors.source.exclude_pod_regex.replace" $excludePodRegex | replace "processors.source.exclude_container_regex.replace" $excludeContainerRegex | replace "processors.source.exclude_host_regex.replace" $excludeHostRegex | replace "processors.resource.cluster.replace" $clusterName | nindent 2 }}
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml:17]: nit: split it on several lines to make reviews easier in the future

Comment (pmm-sumo) [reply deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml:17]: @sumo-drosiek totally agree, tried to split into several lines but must have been doing it wrong and the chart never rendered; do you know the right way to do it?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml:17]: I believe something like that:
```
{{- $yamlFile = replace "processors.source.collector.replace" $_collector $yamlFIle }}
```

**Pull request merged**